evfl/entry_point.py

```diff
@@ -1,15 +1,51 @@
 from evfl.common import StringHolder
 import evfl.event
 from evfl.util import *
+from evfl.dic import DicReader
+from evfl.enums import VariableType
+import os
+
+
+class Variable(BinaryObject):
+    __slots__ = ["num", "type", "value"]
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.type: VariableType
+
+    def _do_read(self, stream: ReadStream) -> None:
+        offset = stream.tell()
+        stream.seek(8, os.SEEK_CUR)
+        self.num = stream.read_u16()
+        self.type = stream.read_u16()
+        assert self.type == VariableType.kInteger or self.type == VariableType.kFloat
+        stream.seek(offset)
+        if self.type == VariableType.kInteger:
+            self.value = stream.read_s32()
+        elif self.type == VariableType.kFloat:
+            self.value = stream.read_f32()
+        stream.seek(12, os.SEEK_CUR)
+
+    def _do_write(self, stream: WriteStream) -> None:
+        if self.type == VariableType.kInteger:
+            stream.write(s32(self.value))
+        if self.type == VariableType.kFloat:
+            stream.write(f32(self.value))
+        stream.write(u32(0))
+        stream.write(u16(self.num))
+        stream.write(u16(self.type))
+        stream.write(u32(0))
+
 
 class EntryPoint(BinaryObject):
-    __slots__ = ['name', 'main_event', '_sub_flow_event_indices', '_sub_flow_event_indices_offset_writer']
+    __slots__ = ['name', 'main_event', '_sub_flow_event_indices', '_sub_flow_event_indices_offset_writer', 'items']
     def __init__(self, name: str) -> None:
         super().__init__()
         self.name = name
         self.main_event: Index[evfl.event.Event] = Index()
         self._sub_flow_event_indices: typing.List[int] = []
         self._sub_flow_event_indices_offset_writer: typing.Optional[PlaceholderWriter] = None
+        self.items = {}
 
     def _do_read(self, stream: ReadStream) -> None:
         sub_flow_event_indices_offset = stream.read_u64()
```

Comment:
can we rename these variables to match the names in https://github.com/open-ead/EventFlow/blob/556793186c3065e7b25a51247d0560d7ff401b81/include/evfl/ResFlowchart.h#L71-L83 ?